data/presets/presets/leisure/marina.json

```diff
@@ -3,24 +3,28 @@
     "fields": [
         "name",
         "operator",
-        "capacity",
+        "phone",
```

Comment:
We try to limit the `fields` array to only a few of the most important fields since some mappers feel like they have to fill out everything. Lots of complex default fields can become intimidating. Please keep the advanced fields under `moreFields`.